CMakeLists.txt

```diff
@@ -156,6 +157,8 @@ if(WITH_SPDK)
   set(HAVE_SPDK TRUE)
 endif(WITH_SPDK)
 
+CHECK_LIBRARY_EXISTS (curl curl_multi_wait "" HAVE_CURL_MULTI_WAIT)
```

Comment:
cmake/modules/Findlibcurl.cmake already has a `CHECK_SYMBOL_EXISTS(curl_multi_wait "curl.h" HAVE_CURL_MULTI_WAIT)`. was it not working for you?